docs/README.md

```diff
@@ -28,6 +28,16 @@
 }
 ```
 
+- When using **typescript**, babel is not required, so your config should look like ([demo](https://github.com/Glavin001/react-hot-ts)):
```

Comment:
Can you move this section to the bottom with its own header? Also, please capitalize TypeScript/Babel.